### PR TITLE
Add bollywood movies

### PR DESCRIPTION
**Pull request by sanjeevkpandit**
¯\\_(ツ)_/¯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,11 +91,13 @@ A list of movies/series which could have really interesting porn parody names.
 | Jab We Met | Jab We Mate |
 | Kabhi Khusi Kabhi Gum | Kabhi Pussy Kabhi Bum |
 | Kuch Kuch Hota Hai | Kuch Kuch Chota Hai |
+| Love Sex Aur Dhokha | :stuck_out_tongue_winking_eye: |
 | Mere Baap Pehle Aap | :stuck_out_tongue_winking_eye: |
 | OMG: Oh My God! | OMG: Oh My Gaand! |
 | Pyaasa | :stuck_out_tongue_winking_eye: <br/> Pyaasi |
 | Rang De Basanti | Ang De Basanti |
 | Sacred Games | Sacred Gays <br/> Secretly Gays |
+| Taarzan - The Wonder Car | Taarzan - The Wonder Man |
```
Comment (aviskarkc10) [inline README.md]: I think we should remove this one?

Comment (aviskarkc10) [reply README.md]: or maybe come up with something for `wonder` as well

Comment (mesaugat) [reply README.md]: Yes, remove. ⛔️

Comment (sanjeevkpandit) [reply README.md]: Removed. @aviskarkc10 @mesaugat

**Pull request merged**